plugins/woocommerce/client/admin/client/dashboard/utils.ts

```diff
@@ -29,19 +29,6 @@ export function getCountryCode( countryState = '' ) {
 	return countryState.split( ':' )[ 0 ];
 }
 
-export function getCurrencyRegion( countryState: string ) {
```

Comment:
Introduced to be used in the old OBW which has since been removed https://github.com/woocommerce/woocommerce-admin/pull/3358